$NetBSD: patch-configure,v 1.3 2022/06/07 18:48:34 adam Exp $

Tweak SEARCH_LIB.
On Darwin, do not run install_name_tool - libs should alredy have correct id.

--- configure.orig	2022-06-06 14:23:45.000000000 +0000
+++ configure
@@ -25384,7 +25384,7 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $wx_cv_std_libfullpath" >&5
 $as_echo "$wx_cv_std_libfullpath" >&6; }
 
-SEARCH_LIB="`echo "$SEARCH_INCLUDE" | sed s@include@$wx_cv_std_libpath@g` $wx_cv_std_libfullpath"
+SEARCH_LIB="`echo "$SEARCH_INCLUDE" | sed s@include@$wx_cv_std_libpath@g` $wx_cv_std_libfullpath @PREFIX@/lib"
 
 if test "$build" != "$host" -a "$GCC" = yes; then
                     if cross_root=`$CC -print-prog-name=ld 2>/dev/null`; then
@@ -33927,7 +33927,6 @@ rm -f core conftest.err conftest.$ac_obj
       *-*-darwin* )
         install_name_tool=`which ${HOST_PREFIX}install_name_tool`
         if test "$install_name_tool" -a -x "$install_name_tool"; then
-            DYLIB_RPATH_POSTLINK="${HOST_PREFIX}install_name_tool -id \$@ \$@"
             cat <<EOF >change-install-names
 #!/bin/sh
 libnames=\`cd lib ; ls -1 | grep '\.[0-9][0-9]*\.dylib\$'\`
@@ -44041,7 +44040,8 @@ fi
       *-*-solaris2* )
         SONAME_FLAG="-h "
         USE_SOVERSION=1
-        USE_SOVERSOLARIS=1
+        USE_SOVERLINUX=1
+        USE_SOTWOSYMLINKS=1
       ;;
 
       *-*-darwin* )
